examples/NonReg/RTC/RTC_Tests/RTC_Tests.ino

```diff
@@ -106,7 +110,11 @@ void loop()
     break;
   }
   Serial.println("Testing asynchronous and synchronous prescaler setting");
+#if defined(STM32F1xx)
```

Comment:
```suggestion
#if defined(STM32F1xx)
Serial.println("For STM32F1 only asynchronous prescaler is used.");
```

Reply:
in case of STM32F1xx
 `Serial.println("Testing only an asynchronous prescaler setting");`